l2geth/eth/backend.go

```diff
@@ -225,6 +225,7 @@ func New(ctx *node.ServiceContext, config *Config) (*Ethereum, error) {
 	eth.miner.SetExtra(makeExtraData(config.Miner.ExtraData))
 
 	log.Info("Backend Config", "max-calldata-size", config.Rollup.MaxCallDataSize, "gas-limit", config.Rollup.GasLimit, "is-verifier", config.Rollup.IsVerifier, "using-bvm", rcfg.UsingBVM)
+
```

Comment:
We can just revert the change to this file

Reply:
> We can just revert the change to this file

reverted.